### PR TITLE
Cleanup and normalize the codebase

### PR DESCRIPTION
**Pull request by bitbager**
Hi @lchrusciel!

I've seen that there are a few coding standards applied to the plugin. I cleaned up it a little. Such great plugin deserves a nice CS 🙂 

Bests,
Nick

### CODE REVIEW

**Comment by lchrusciel** [comment]
Thank you very much Mikołaj, but I would prefer to add some EasyCodingStandard also so we can keep the code clean :)

**Comment by bitbager** [comment]
I will clean it up today :)

File: src/Command/PutOptionBasedConfigurableItemToCart.php [hunk 1]
```diff
@@ -48,33 +40,21 @@ public function __construct($orderToken, $product, array $options, $quantity)
...
-     */
     public function orderToken()
     {
         return $this->orderToken;
     }
 
-    /**
-     * @return string
-     */
     public function product()
```
Comment (bitbager) [inline src/Command/PutOptionBasedConfigurableItemToCart.php]: @lchrusciel, shouldn't these methods also include return type?

Comment (lchrusciel) [reply src/Command/PutOptionBasedConfigurableItemToCart.php]: That's why we need ECS and PHPStan together

File: src/Command/RemoveCoupon.php [hunk 2]
```diff
@@ -8,24 +8,16 @@
...
-    /**
-     * @var string
-     */
+    /** @var string */
     private $orderToken;
 
-    /**
-     * @param string $orderToken
-     */
     public function __construct($orderToken)
```
Comment (lchrusciel) [inline src/Command/RemoveCoupon.php]: Also here we should add typehint

File: src/Command/AddCoupon.php [hunk 3]
```diff
@@ -8,38 +8,24 @@
...
+    /** @var string */
     private $couponCode;
 
-    /**
-     * @param string $orderToken
-     * @param string $couponCode
-     */
     public function __construct($orderToken, $couponCode)
     {
         Assert::allString([$orderToken, $couponCode]);
```
Comment (lchrusciel) [inline src/Command/AddCoupon.php]: Every assertion from the constructor should be moved to scalar type hints(if it is possible thus)

**Comment by lchrusciel** [comment]
Closing in favour of #307

**Pull request closed without merging**